README.md

```diff
@@ -1,3 +1,74 @@
-# spark-android
+# Spark Android Design System
+
+<p align="center">
+<picture>
+    <source media="(prefers-color-scheme: dark)" srcset="art/spark-logo-dark.svg">
+    <img alt="Spark Design System logo" src="art/spark-logo-light.svg">
+  </picture>
+</p>
 
 [![👷 Build → 🧑‍🔬 Test → 🕵️ Lint](https://github.com/adevinta/spark-android/actions/workflows/ci.yml/badge.svg)](https://github.com/adevinta/spark-android/actions/workflows/ci.yml)
+
+Compose Spark Design System is based on Material 3 compose artifact described
```

Comment:
Do we really want to name it `Compose Spark Design System`? and not just `Spark Design System` ?

```suggestion
Spark Design System is based on Material 3 compose artifact described
```

Reply:
I've talked with iOS since they'll support UI Kit and SwiftUI and they won't specify the tech it's based on 👍